dawn/renderer/reducers/info.js

```diff
@@ -28,7 +28,6 @@ const info = (state = {}, action) => {
     case 'UPDATE_STATUS':
       return {
         ...state,
-        isRunningCode: (action.status.value === 1),
```

Comment:
I assume that sometime in the future this functionality will need to be added back right? We will need some way of keeping track of if code is running or not eventually I assume.